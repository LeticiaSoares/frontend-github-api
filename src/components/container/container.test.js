import React from 'react'
import { shallow } from 'enzyme';
import Container from './index'
import 'jest-styled-components'

describe('<Container />', () => {
    it('should match default snapshot', () => {
        const component = shallow(<Container />)
        expect(component).toMatchSnapshot()
    })
    it('should have the right displayName', () => {
        expect(Container.displayName).toEqual('Container')
    })
})
