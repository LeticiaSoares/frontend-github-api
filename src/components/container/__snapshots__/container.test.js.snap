// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Container /> should match default snapshot 1`] = `ShallowWrapper {}`;
